monitoring/grafana/dashboards/jsonnet/grafana_dashboards.jsonnet

```diff
@@ -0,0 +1,42 @@
+local grafana = import '../../../../grafonnet-lib/grafonnet/grafana.libsonnet';
```

Comment:
@tchaikov I am getting an error in the make-check 'couldn't open import "../../../../grafonnet-lib/grafonnet/grafana.libsonnet": no match locally or in the Jsonnet library paths.'. Maybe am doing something wrong here as the jsonnet file is not being able to locate the grafonnet-lib directory. I used `include_directories(grafonnet-lib ${CMAKE_BINARY_DIR}})` and tried `add_dependencies` as well. Can you please suggest here some solution.